packages/db/src/search/queries.ts

```diff
@@ -243,9 +243,10 @@ export const convertQueryStringToDbQueryParameters = (
 
   const dbQueryParameters: DbQueryParameters = {};
   dbQueryParameters.page = Number.parseInt(page ?? '1', 10);
-  dbQueryParameters.limit = Number.parseInt(limit ?? '10', 10);
-  dbQueryParameters.offset = (dbQueryParameters.page - 1) * dbQueryParameters.limit;
-
+  if (limit !== null) {
```

Comment:
is it intended that this be:
null = unlimited
undefined = 10
? having the different null vs undefined behavior seems weird if there isn't a direct desire for it

Reply:
That's more or less the case - we want a default if it's not defined, null is explicitly setting 'no limit', and feels better as an overload than '0' or the like.  We could also add another queryParam instead if that feels bad.

Reply:
I _think_ that makes sense. If you don't include `limit`, the default is 10. If you explicitly say limit = null, the limit is removed. It is an interesting thing. I think it makes sense.